modules/installation-aws-upi-bootstrap.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-aws-upi-bootstrap-{context}"]
+= Initializing the bootstrap node on AWS UPI
+
+After you create all of the required infrastructure in Amazon Web Services (AWS),
+you can install the cluster.
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the Ignition config files for your cluster.
```

Comment:
We tend to spell out "configuration files" instead of just saying "config". I see this usage throughout the files in this PR, so we should probably pick a way. I don't see any guidance for our team in how we'll treat this in 4.x docs, unless I missed it somewhere.

Reply:
I know. :/ The references in the CoreOS docs are to "Ignition configs" more often than not, and neither Chris nor I have ever heard anyone refer to them as "Ignition configuration." I think we might need another exception, like the always-lowercase "etcd."

Reply:
I had a chat with Vikram, and I'm adding "Ignition config" to the glossary. https://github.com/openshift/openshift-docs/pull/14461